# COGS 108 - Final Project Proposal

# Names

- Leica Shen
- Yuxuan Wu
- Eddie Ho
- Xuewen Yang

# Research Question

How has the COVID-19 pandemic impact global warming? Specifically, how do changes in greenhouse gas emissions during 1/30/2020 - 4/11/2023 resulting from pandemic-related lockdowns and reduced economic activity impact the rate of global temperature increase and sea level rise?

## Background and Prior Work


- Include a general introduction to your topic
- Include explanation of what work has been done previously
- Include citations or links to previous work

This section will present the background and context of your topic and question in a few paragraphs. Include a general introduction to your topic and then describe what information you currently know about the topic after doing your initial research. Include references to other projects who have asked similar questions or approached similar problems. Explain what others have learned in their projects.

Find some relevant prior work, and reference those sources, summarizing what each did and what they learned. Even if you think you have a totally novel question, find the most similar prior work that you can and discuss how it relates to your project.

References can be research publications, but they need not be. Blogs, GitHub repositories, company websites, etc., are all viable references if they are relevant to your project. It must be clear which information comes from which references. (2-3 paragraphs, including at least 2 references)

 **Use inline citation through Markdown footnotes to specify which references support which statements** 

For example: After government genocide in the 20th century, real birds were replaced with surveillance drones designed to look just like birds[^lorenz]. 
Use a minimum of 2 or 3 citations, but we prefer more[^admonish]. You need enough to fully explain and back up important facts. 

[^lorenz]: Lorenz, T. (9 Dec 2021) Birds Aren’t Real, or Are They? Inside a Gen Z Conspiracy Theory. *The New York Times*. https://www.nytimes.com/2021/12/09/technology/birds-arent-real-gen-z-misinformation.html 
[^admonish]: Also refs should be important to the background, not some randomly chosen vaguely related stuff. Include a web link if possible in refs as above.


# Hypothesis



- Include your team's hypothesis
- Ensure that this hypothesis is clear to readers
- Explain why you think this will be the outcome (what was your thinking?)

What is your main hypothesis/predictions about what the answer to your question is? Briefly explain your thinking. (2-3 sentences)

# Data

**Ideal dataset**:
* **Global temperature data**: Accurate and reliable measurements of monthly average or annual average temperatures for different regions of the world, recorded over decades that include the period from January 30, 2020 to November 4, 2023.

* **Greenhouse gas emissions data**: Accurate and reliable measurements of global greenhouse gas emissions, including records of carbon dioxide (CO2), methane (CH4), and nitrous oxide (N2O), over decades that include the period from January 30, 2020 to November 4, 2023.

* **Sea-level rise data**: Accurate and reliable measurements of global sea-level rise over decades that include the period from January 30, 2020 to November 4, 2023.

* **Data sources**: The greenhouse gas emissions data could be obtained from industry reports, national and international agencies such as *The Emissions Database for Global Atmospheric Research*, etc. Temperature and sea-level rise data could be obtained from meteorological and oceanographic stations, etc.

* **Data storage/organization**: The data should be stored in a structured format such as *.CSV* format. We can store the data in Pandas DataFrame for better reference. 

* **Sample size**: Includes data on at least the last 50 years. Within each year, there may be multiple data points for each month, so a good sample size will be 1000. 

**Real dataset**:

In [1]:
import pandas as pd
import numpy as np

In [16]:
methane_final = pd.read_csv('data/Methane_final.csv')
methane_final = methane_final.drop(columns='Unnamed: 0')
methane_final.head()

region  country   emissions         type                           segment   
0  Africa  Algeria  257.611206  Agriculture                             Total  \
1  Africa  Algeria    0.052000       Energy                         Bioenergy   
2  Africa  Algeria  130.798996       Energy  Gas pipelines and LNG facilities   
3  Africa  Algeria   69.741898       Energy  Gas pipelines and LNG facilities   
4  Africa  Algeria  213.987000       Energy                       Onshore gas   

     reason   baseYear                                              notes  
0       All  2019-2021  Average based on United Nations Framework Conv...  
1       All       2022  Estimates from end-uses are for 2020 or 2021 (...  
2  Fugitive       2022                                      Not available  
3    Vented       2022                                      Not available  
4  Fugitive       2022                                      Not available

The *methane_final* dataset contains methane gas emissions data around the world for 2019 - 2022. It differs with our ideal dataset on that it does not contain data before 2019 and data after 2022. It also does not contain gas emissions data for other kinds of greenhouse gas.

In [18]:
sea_level = pd.read_csv('data/sea_levels_2015.csv')
sea_level.head()

Time   GMSL  GMSL uncertainty
0  1880-01-15 -183.0              24.2
1  1880-02-15 -171.1              24.2
2  1880-03-15 -164.3              24.2
3  1880-04-15 -158.2              24.2
4  1880-05-15 -158.7              24.2

The *sea_level* dataset contains global average absolute sea level changing data from 1880 to 2014. It does not contains the data for 2015 and beyond. Thus, we still need to look for another dataset to merge with. 

# Ethics & Privacy

- Thoughtful discussion of ethical concerns included
- Ethical concerns consider the whole data science process (question asked, data collected, data being used, the bias in data, analysis, post-analysis, etc.)
- How your group handled bias/ethical concerns clearly described

Acknowledge and address any ethics & privacy related issues of your question(s), proposed dataset(s), and/or analyses. Use the information provided in lecture to guide your group discussion and thinking. If you need further guidance, check out [Deon's Ethics Checklist](http://deon.drivendata.org/#data-science-ethics-checklist). In particular:

- Are there any biases/privacy/terms of use issues with the data you propsed?
- Are there potential biases in your dataset(s), in terms of who it composes, and how it was collected, that may be problematic in terms of it allowing for equitable analysis? (For example, does your data exclude particular populations, or is it likely to reflect particular human biases in a way that could be a problem?)
- How will you set out to detect these specific biases before, during, and after/when communicating your analysis?
- Are there any other issues related to your topic area, data, and/or analyses that are potentially problematic in terms of data privacy and equitable impact?
- How will you handle issues you identified?

# Team Expectations 


Read over the [COGS108 Team Policies](https://github.com/COGS108/Projects/blob/master/COGS108_TeamPolicies.md) individually. Then, include your group’s expectations of one another for successful completion of your COGS108 project below. Discuss and agree on what all of your expectations are. Discuss how your team will communicate throughout the quarter and consider how you will communicate respectfully should conflicts arise. By including each member’s name above and by adding their name to the submission, you are indicating that you have read the COGS108 Team Policies, accept your team’s expectations below, and have every intention to fulfill them. These expectations are for your team’s use and benefit — they won’t be graded for their details.

* *Team Expectation 1*
* *Team Expectation 2*
* *Team Expecation 3*
* ...

# Project Timeline Proposal

Specify your team's specific project timeline. An example timeline has been provided. Changes the dates, times, names, and details to fit your group's plan.

If you think you will need any special resources or training outside what we have covered in COGS 108 to solve your problem, then your proposal should state these clearly. For example, if you have selected a problem that involves implementing multiple neural networks, please state this so we can make sure you know what you’re doing and so we can point you to resources you will need to implement your project. Note that you are not required to use outside methods.



| Meeting Date  | Meeting Time| Completed Before Meeting  | Discuss at Meeting |
|---|---|---|---|
| 1/20  |  1 PM | Read & Think about COGS 108 expectations; brainstorm topics/questions  | Determine best form of communication; Discuss and decide on final project topic; discuss hypothesis; begin background research | 
| 1/26  |  10 AM |  Do background research on topic | Discuss ideal dataset(s) and ethics; draft project proposal | 
| 2/1  | 10 AM  | Edit, finalize, and submit proposal; Search for datasets  | Discuss Wrangling and possible analytical approaches; Assign group members to lead each specific part   |
| 2/14  | 6 PM  | Import & Wrangle Data (Ant Man); EDA (Hulk) | Review/Edit wrangling/EDA; Discuss Analysis Plan   |
| 2/23  | 12 PM  | Finalize wrangling/EDA; Begin Analysis (Iron Man; Thor) | Discuss/edit Analysis; Complete project check-in |
| 3/13  | 12 PM  | Complete analysis; Draft results/conclusion/discussion (Wasp)| Discuss/edit full project |
| 3/19  | Before 11:59 PM  | NA | Turn in Final Project & Group Project Surveys |